bob/src/hw_metrics_collector.rs

```diff
@@ -124,11 +129,14 @@ impl HWMetricsCollector {
         }
     }
 
-    fn update_space_metrics_from_disks(disks: &HashMap<PathBuf, DiskName>) -> DiskSpaceMetrics {
+    fn update_space_metrics_from_disks(
+        disks: &HashMap<PathBuf, DiskName>,
+    ) -> HashMap<DiskName, DiskSpaceMetrics> {
         let disks_metrics = Self::space(disks);
-        gauge!(TOTAL_SPACE, bytes_to_mb(disks_metrics.total_space) as f64);
-        gauge!(USED_SPACE, bytes_to_mb(disks_metrics.used_space) as f64);
-        gauge!(FREE_SPACE, bytes_to_mb(disks_metrics.free_space) as f64);
+        let summed_space: DiskSpaceMetrics = disks_metrics.values().sum();
```

Comment:
Unfortunately, this sum is incorrect, because multiple bob disks can be mapped to a single physical disk

Reply:
I suggest to return a map from `space` function, which key is a mount point (not DiskName) and DiskName can be a field of `DiskSpaceMetrics` structure

Reply:
Mount point as a key is even worse alternative to the `DiskName` because there are usually multiple mount points for the same Bob disk (but this may be changed in #828). The best way to ensure the uniqueness of physical disks is to map them to the `f_fsid` of `statvfs` system call. In that way we don't depend on any outer function to provide correct map of mount points and disk names and ensure the uniqueness of the result

Reply:
Although I'm not sure that f_fsid is unique across multiple physical disks...